.github/workflows/publish_pypi.yml

```diff
@@ -0,0 +1,47 @@
+name: Publish to Pypi
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version upload to pypi'
+        required: true
+      pypi_repo:
+        description: 'Repo to upload to (testpypi or pypi)'
+        default: 'testpypi'
+        required: true
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-python@v2
+      with:
+        python-version: '3.8'
+    - name: Install dependencies
+      run: |
+        pip install -U wheelhouse_uploader pyyaml
+    - name: Downloading wheels and sdist from staging
+      run: |
+        echo "Download ${{ github.event.inputs.version }} wheels and sdist"
```

Comment:
Even those this workflow can only be triggered by project members, I think we should not use sanitized users'  input in bash script directly (`${{ github.event.inputs.version }}` in this case). Let's assign it to an environment variable first.